Wikipedia/Categories/UIView+WMFDefaultNib.m

```diff
@@ -1,17 +1,15 @@
-//
-//  UIView+WMFDefaultNib.m
-//  Wikipedia
-//
-//  Created by Brian Gerstle on 3/6/15.
-//  Copyright (c) 2015 Wikimedia Foundation. All rights reserved.
-//
 
 #import "UIView+WMFDefaultNib.h"
+#import "NSString+Extras.h"
 
 @implementation UIView (WMFDefaultNib)
 
 + (NSString*)wmf_nibName {
-    return NSStringFromClass(self);
+    /* Swift has "Namespaced" class names that prepend the module
```

Comment:
Man, harry is such a jerk.  :stuck_out_tongue: 

Yeah, if we ever use Swift reusable views in another module that doesn't define ObjC names... i guess this is fine then